lib/rdkafka/consumer.rb

```diff
@@ -393,7 +419,7 @@ def commit(list=nil, async=false)
     #
     # @return [Message, nil] A message or nil if there was no new message within the timeout
     def poll(timeout_ms)
-      return unless @native_kafka
+      closed_consumer_check(__method__)
```

Comment:
This is a change in behaviour, but I'm curious if we really want to allow a `#poll` when the underlying kafka instance is closed?

Reply:
I think raising an exception is fine in that case.